tests/BackwardsCompatiblePayloadSerializerFactoryTest.php

```diff
@@ -82,7 +82,15 @@ public function it_handles_main_language(
         /** @var EventCreated|PlaceCreated|OrganizerCreatedWithUniqueWebsite $created */
         $created = $this->serializer->deserialize($decoded);
 
-        $this->assertEquals($expectedMainLanguage, $created->getMainLanguage());
+        /*
+         * The signature of getMainLanguage() is not consistent, sometimes this returns a string, sometimes a Language object
```

Comment:
This is because the new approach for events is to only use primitive types without validation. Otherwise, if we take the strict validation from the JSON validation or the VO it could lead to runtime exceptions when reading the event stream of older resources.